analysis_tasks/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM r-base:3.2.5
+
+RUN R --slave --silent --vanilla \
+    -e 'install.packages(c("optparse", "caret", "plyr", "gbm", "doParallel"), repos="http://cran.rstudio.com")'
+
+RUN mkdir -p /opt/analysis_tasks
+WORKDIR /opt/analysis_tasks
+
+COPY . /opt/analysis_tasks
+
+# Provide access to the R environment by default if run without parameters
+CMD ["R", "--vanilla"]
```

Comment:
I initially had it that way, but my understanding of the way `ENTRYPOINT` works is that it would make it more difficult to run the container with `R` because it would run `Rscript R <args>` by default, so I was thinking it would be easier to use for both debugging and non-interactive tasks if it didn't define a custom entrypoint. It'll also make it a bit clearer what's happening when a container is run since the full command won't be split across two files and there are at least two reasonable default choices in this case.